azkaban-webserver/src/main/resources/azkaban/webapp/servlet/velocity/projectsidebar.vm

```diff
@@ -16,7 +16,7 @@
 
           <div class="well" id="project-sidebar">
             <h3>$project.name</h3>
-            <p class="editable" id="project-description">$project.description</p>
+            <p class="editable" id="project-description">$esc.html($project.description)</p>
```

Comment:
just a side question,  do we have a way to turn escaping on at template level ?  if that is possible it will be really nice that we don't need to worry for all the properties.